experimental/add_to_app/android_using_plugin/app/src/main/java/dev/flutter/example/androidusingplugin/MyApplication.kt

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package dev.flutter.example.androidusingplugin
+
+import android.app.Application
+import dev.flutter.plugins.GeneratedPluginRegistrant
+import io.flutter.embedding.engine.FlutterEngine
+import io.flutter.embedding.engine.FlutterEngineCache
+import io.flutter.embedding.engine.dart.DartExecutor
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.view.FlutterMain
+
+const val ENGINE_ID = "1"
+
+class MyApplication : Application() {
+    var count = 0
+
+    private var channel: MethodChannel? = null
+
+    override fun onCreate() {
+        super.onCreate()
+
+        FlutterMain.startInitialization(this)
```

Comment:
Are you targeting stable at this point? or master? because I think you can remove these lines on master (worth running and verifying)

Reply:
Interesting. I tried removing those lines a few days ago, and got an exception. Either changing the `.metadata` file for the projects fixed it, or something landed in master since your PR was merged.

Is there a delay between a merge in the engine repo and when that code shows actually up in the SDK's master channel?

Reply:
Good question. Yeah the engine version has to be incremented in the master channel for the framework. This tends to happen every few days. I would say maybe give it one more try. If the latest framework still doesn't have the change then this is fine.